install_config/install/advanced_install.adoc

```diff
@@ -164,123 +164,147 @@ ifdef::openshift-enterprise[]
+master may require access. Defaults to `10.128.0.0/14` and cannot be arbitrarily
 re-configured after deployment, although certain changes to it can be made in
 the xref:../configuring_sdn.adoc#configuring-the-pod-network-on-masters[SDN
 master configuration].
 
-|`*osm_host_subnet_length*`
+|`osm_host_subnet_length`
 |This variable specifies the size of the per host subnet allocated for pod IPs
 by
 xref:../../architecture/additional_concepts/sdn.adoc#sdn-design-on-masters[{product-title}
-SDN]. Defaults to *9* which means that a subnet of size /23 is allocated to each
+SDN]. Defaults to `9` which means that a subnet of size /23 is allocated to each
 host; for example, given the default 10.128.0.0/14 cluster network, this will
-allocate 10.128.0.0/23, 10.128.2.0/23, 10.128.4.0/23, and so on. This *cannot* be
+allocate 10.128.0.0/23, 10.128.2.0/23, 10.128.4.0/23, and so on. This cannot be
 re-configured after deployment.
 
-|`*openshift_use_flannel*`
+|`openshift_use_flannel`
 |This variable enables *flannel* as an alternative networking layer instead of
 the default SDN. If enabling *flannel*, disable the default SDN with the
-*openshift_use_openshift_sdn* variable. For more information, see xref:../configuring_sdn.adoc#using-flannel[Using Flannel].
+`openshift_use_openshift_sdn` variable. For more information, see xref:../configuring_sdn.adoc#using-flannel[Using Flannel].
 
-|`*openshift_docker_additional_registries*`
+|`openshift_docker_additional_registries`
 |{product-title} adds the specified additional registry or registries to the
-Docker configuration.
+*docker* configuration.
 
-|`*openshift_docker_insecure_registries*`
+|`openshift_docker_insecure_registries`
 |{product-title} adds the specified additional insecure registry or registries
-to the Docker configuration.
+to the *docker* configuration.
 
-|`*openshift_docker_blocked_registries*`
-|{product-title} adds the specified blocked registry or registries to the Docker
+|`openshift_docker_blocked_registries`
+|{product-title} adds the specified blocked registry or registries to the *docker*
 configuration.
 
-|`*openshift_hosted_metrics_public_url*`
-|This variable sets the host name for integration with the metrics console. The
-default is
-`\https://hawkular-metrics.{{openshift_master_default_subdomain}}/hawkular/metrics`
+|`openshift_hosted_metrics_public_url`
+|This variable sets the host name for integration with the metrics console by
+overriding `metricsPublicURL` in the master configuration for cluster metrics.
 If you alter this variable, ensure the host name is accessible via your router.
+<<<<<<< HEAD
+=======
+See xref:advanced-install-cluster-metrics[Configuring Cluster Metrics] for
+details.
+
+|`openshift_template_service_broker_namespaces`
+|This variable enables the template service broker by specifying one of more
+namespaces whose templates will be served by the broker.
+>>>>>>> c31db5f... Adjust openshift_hosted_metrics_public_url info / formatting
```

Comment:
@gaurav-nelson Oopsie, some merge conflict stuff got in here. I'll do a follow-up now to smooth it out.